src/app/api/users/[id]/route.ts

```
@@ -0,0 +1,101 @@
+            }
+        });
+
+        if (!existingUser) {
+            return NextResponse.json(
+                { error: "User not found" },
+                { status: 404 }
+            );
+        }
+
+        // Prepare update data
+        const updateData: any = {};
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)